rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -238,6 +234,7 @@ components:
       description: Version number.
       type: string
       minLength: 3
+      maxLength: 80
```

Comment:
Is this arbitrary? Same question for the other max values

Reply:
Some of them ... yup. Some of them are based on typical sizes supported out there by existing servers. So, as always, it depends :-)